applications/serial_lte_modem/Kconfig

```diff
@@ -166,6 +166,18 @@ config SLM_PPP
 	  when the LTE link (default PDN) is connected/disconnected,
 	  regardless of what is done with AT#XPPP.
 
+if SLM_PPP
+
+config SLM_PPP_FALLBACK_MTU
+	int "Fallback MTU to be used by PPP"
+	default 1280
```

Comment:
Could this have maximum range of 1500, instead of build assert? 

Also, is there any minimum range for this?

Reply:
These ranges usually bite you later on...

You absolutely need a build time assert to ensure buffers are big enough... Ranges cannot depend on another configurable values. So I like it being as simple as this.